.buildkite/pipeline.yml

```diff
@@ -10,76 +10,84 @@ env:
   IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
 
 steps:
-  - label: ":go: Run check-static"
```

Comment:
I would restore this file before merging, in case it is needed to do more tests.